plinth/service.py

```diff
@@ -45,7 +45,8 @@ class Service(object):
     - is_running (optional): Boolean or a method returning Boolean
     """
     def __init__(self, service_id, name, ports=None, is_external=False,
-                 is_enabled=None, enable=None, disable=None, is_running=None):
+                 is_enabled=None, enable=None, disable=None, is_running=None,
+                 allow_uninstall=True):
```

Comment:
allow_uninstall should not be property of a service but property of an application.  This is similar to how list of packages is itself a property of application instead of a property of a services.  The difference is that:
- Application has several services and all of them are installed and removed in an single step.
- Package may not be associated with any service.